src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/crushmap/crushmap.component.scss

```diff
@@ -1,7 +1,3 @@
-::ng-deep cd-crushmap tree-root {
-  tree-viewport {
-    div:first-child {
-      height: unset !important;
-    }
-  }
+.tree-container {
+  height: calc(100vh - 180px);
```

Comment:
I would suggest to use for a better look. In this case the bottom border is the same as to the left and right.
```suggestion
  height: calc(100vh - 200px);
```
![Bildschirmfoto vom 2020-12-07 15-44-07](https://user-images.githubusercontent.com/1897962/101364712-28c8b900-38a3-11eb-9ea6-1a6b95b0d0e7.png)

But finally IMO a solution without a hardcoded pixel number would be the best.

Reply:
I think it is hard if not impossible without a hardcoded pixel number. I've played with flexbox, but cannot make it work. It seems that I still need to specify the height somewhere in `px` or `vh`. Maybe I can make `body` to be `100vh` height, but that's a big change. I think even if it is possible, it is not worthful given such a simple solution works fine.

I use 180px because I want to give users the largest possible viewport. But I will change it as you wish.